tests/System.Memory.Polyfill.Tests/System.Memory.Polyfill.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;netcoreapp3.0</TargetFrameworks>
```

Comment:
We don't install any runtimes when bootstrapping, which we could by adding an entry to `global.json`, however, I don't think it makes sense to test against an older framework.